ui/app/mainui/AppMain.qml

```diff
@@ -998,11 +998,17 @@ Item {
                         }
 
                         StatusAction {
-                            text: qsTr("View Community")
-                            icon.name: "group-chat"
+                            text: qsTr("Community Info")
+                            icon.name: "info"
                             onTriggered: popups.openCommunityProfilePopup(appMain.rootStore, model, communityContextMenu.chatCommunitySectionModule)
                         }
 
+                        StatusAction {
+                            text: qsTr("Community Rules")
+                            icon.name: "text"
+                            onTriggered: popups.openCommunityRulesPopup(model)
```

Comment:
Passing the model to a popup can result in crashing the app on model change.
There are two alternatives here:
1. Pass to the popup all the needed info (name, image, color, intro) if we don't care about the live updates for these properties. Something like `onTriggered: popups.openCommunityRulesPopup(model.name, model.image, model.color, model.introText)`

2. Pass to the popup the community id and extract the info in the popup. With this approach all the properties are live.

Reply:
I didn't know about that potential crash. I fixed it by doing the first solution. It's not the end that this section doesn't update.
Thanks @alexjba